ngraph/frontend/paddlepaddle/src/op/yolo_box.cpp

```diff
@@ -0,0 +1,262 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <limits>       // std::numeric_limits
+#include <numeric>
+
+#include <ngraph/opsets/opset6.hpp>
+#include "yolo_box.hpp"
+
+namespace ngraph {
+namespace frontend {
+namespace pdpd {
+namespace op {
+    using namespace opset6;
+    using namespace element;
+
+NamedOutputs yolo_box (const NodeContext& node_context) {
+    auto data = node_context.get_ng_input("X");
+    auto image_size = node_context.get_ng_input("ImgSize");
+
+    auto input_shape = data.get_partial_shape();
+    uint32_t input_height = input_shape[2].get_length();
+    uint32_t input_width = input_shape[3].get_length();   
```

Comment:
Same here

Reply:
@ceciliapeng2011 Could you use shape_of to get shape ?

Reply:
TODO: dynamic shape support CVS-55264